src/main/java/io/beanmapper/config/Configuration.java

```diff
@@ -80,20 +80,73 @@
 
     List<BeanConverter> getBeanConverters();
 
+    /**
+     * Returns the entire list of strict bean pairs. The properties on the strict side must
+     * have matching properties on the other, non-strict side.
+     * @return the entire list of strict bean pairs.
+     */
+    List<BeanPair> getBeanPairs();
+
     Boolean isConverterChoosable();
 
     void withoutDefaultConverters();
 
     /**
+     * Returns the classname suffix that determines a source class is to be treated as strict
+     * with regards to mapping. Default is "Form"
+     * @return the source classname suffix for a class to be treated as strict
+     */
+    String getStrictSourceSuffix();
+
+    /**
+     * Returns the classname suffix that determines a target class is to be treated as strict
+     * with regards to mapping. Default is "Result"
+     * @return the target classname suffix for a class to be treated as strict
+     */
+    String getStrictTargetSuffix();
+
+    /**
+     * Determines if strict mapping convention will be applied. This means that if a source
+     * class has the strict source suffix, or a target class has the strict target suffix,
+     * the classes will be treated as if they are strict. This implies that all of their
+     * properties will require matching properties on the other side. Default is true.
+     * @return if true, the strict mapping convention will be applied
+     */
+    Boolean isApplyStrictMappingConvention();
+
+    /**
+     * Returns the collection of strictSourceSuffix, strictTargetSuffix and
+     * applyStrictMappingConvention properties.
+     * @return all properties required for dealing with the strict mapping convention
+     */
+    StrictMappingConfiguration getStrictMappingConfiguration();
```

Comment:
Deze zou ik getStrictMappingProperties() noemen. De method zit al in een class die Configuration heet...